src/PIL/ImageOps.py

```diff
@@ -236,6 +236,34 @@ def colorize(image, black, white, mid=None, blackpoint=0, whitepoint=255, midpoi
     return _lut(image, red + green + blue)
 
 
+def contain(image, size, method=Image.BICUBIC):
+    """
+    Returns a sized version of the image, expanded to fill the requested aspect ratio
+    and size.
```

Comment:
Maybe something like this is clearer?
```suggestion
    Returns a resized version of the image, expanded to fill the requested size by cropping, and with no padding.
```

Reply:
There isn't any cropping in this method.
I've pushed a commit with
> Returns a resized version of the image, set to the maximum width and height within the requested size, while maintaining the original aspect ratio.

Feel free to suggest further changes.

Reply:
Sounds good, thanks! (I forgot hopper.png is square when looking at the examples.)